106 - Enable django-linear-migrations

earthcomfy opened the pull request:
Closes: #106

This PR is to configure `django-linear-migrations` so that linear migration history can be ensured.

brylie reviewed (changes requested):
Please resolve conflicts in `poetry.lock`

See: https://www.peterbe.com/plog/how-to-resolve-a-git-conflict-in-poetry.lock

earthcomfy commented:
@brylie After I updated my branch, I saw that a custom migration command was created. Quoting `django_linear_migrations`:
> The app relies on overriding the built-in makemigrations command. If your project has a custom makemigrations command, ensure the app containing your custom command is above django_linear_migrations, and that your command subclasses its Command class:

Thus I've updated the custom migration command.

brylie commented:
@all-contributors please add @earthcomfy for code
